pkgs/development/python-modules/python-toolbox/default.nix

```diff
@@ -4,23 +4,25 @@
 , fetchFromGitHub
 , isPy27
 , nose
+, pytest
 }:
 
 buildPythonPackage rec {
-  version = "0.9.4";
+  version = "1.0.10";
   pname = "python_toolbox";
   disabled = isPy27;
 
   src = fetchFromGitHub {
     owner = "cool-RR";
     repo = pname;
     rev = version;
-    sha256 = "1qy2sfqfrkgxixmd22v5lkrdykdfiymsd2s3xa7ndlvg084cgj6r";
+    sha256 = "1hpls1hwisdjx1g15cq052bdn9fvh43r120llws8bvgvj9ivnaha";
   };
 
   checkInputs = [
     docutils
     nose
```

Comment:
I think nose is no longer necessary
```suggestion
```

Reply:
Yep, removed. Thx.
(I wonder if we can determine such things automatically via checking reads of the dependencies etc.)